apps/studio/src/constants/formBuilder.ts

```diff
@@ -12,4 +12,5 @@ export const JSON_FORMS_RANKING = {
   RadioControl: 3,
   GroupLayoutRenderer: 1,
   VerticalLayoutRenderer: 1,
+  Catchall: 99999999999
```

Comment:
Why do we even have sectionIdx?

Reply:
It's for compat with the classic infobar infocols infocards components?

Reply:
Yeap! We remove the props in another PR but I reopened this because I wanted to check if there's value in a default case to catch fall throughs